### PR TITLE
Add 1 Project

### PR DESCRIPTION
**Pull request by AshanthaLahiru**
JAVA ## Decimal To Binary Using Stack

### CODE REVIEW

File: JAVA/Decimal To Binary Using Stack/DecimalToBinary.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        else {
+            Node old = top;
+            top=new Node(i);
+            top.next=old;
+        }
+
+    }
+    
+    public Node pop () {
+        Node removed=top;
```
Comment (amanmehara) [inline JAVA/Decimal To Binary Using Stack/DecimalToBinary.java]: What if top is null?
i.e. stack is empty?

This should be handled.

File: JAVA/Decimal To Binary Using Stack/DecimalToBinary.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import java.util.Scanner;
+
+public class DecimalToBinary {
+
+    public static void main(String[] args) {
+        
+        Stack stk= new Stack();
+        Scanner scn = new Scanner(System.in);
+        
+        System.out.println("Wellcome to Programme - Decimal To Binary Using Stack \n\nEnter a Decimal Number : ");
```
Comment (amanmehara) [inline JAVA/Decimal To Binary Using Stack/DecimalToBinary.java]: Typo. 😛

File: JAVA/Decimal To Binary Using Stack/DecimalToBinary.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        
+        System.out.println("Wellcome to Programme - Decimal To Binary Using Stack \n\nEnter a Decimal Number : ");
+        int number = scn.nextInt(); // Getting an integer from user
+        
+        for(int x = number ; x != 0; x = x/2) {
+            stk.push(x%2);
+        }
+        System.out.print("Binary number equal to " + number + " : ");
+        while(!stk.isEmpty()) {
+            System.out.print(stk.pop().data);
```
Comment (amanmehara) [inline JAVA/Decimal To Binary Using Stack/DecimalToBinary.java]: This not empty check should be with the stack DS itself.

Comment (AshanthaLahiru) [reply JAVA/Decimal To Binary Using Stack/DecimalToBinary.java]: What do you mean?

Comment (amanmehara) [reply JAVA/Decimal To Binary Using Stack/DecimalToBinary.java]: The pop() should also check if stack is empty or not.

This condition here is ok.

**Pull request merged**